src/WebJobs.Script.Grpc/Abstractions/WorkerConfig.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public class WorkerConfig
+    {
+        public string ExecutablePath { get; set; }
+
+        public List<string> ExecutableArguments { get; set; } = new List<string>();
+
+        public List<string> WorkerArguments { get; set; } = new List<string>();
+
+        public string WorkerPath { get; set; }
```

Comment:
Xml doc for all these new types would help me understand these properties better. I'll wait for you to add the doc before I question about what these all are :)